source/screens/caseScreens/CaseSummary.js

```diff
@@ -1,21 +1,36 @@
-import React, { useContext, useEffect, useState, useRef, useCallback } from 'react';
-import { View, Animated, Easing, ScrollView } from 'react-native';
-import PropTypes from 'prop-types';
-import { CaseState } from 'app/store/CaseContext';
-import FormContext from 'app/store/FormContext';
-import styled from 'styled-components/native';
-import { useIsFocused } from '@react-navigation/native';
-import icons from '../../helpers/Icons';
-import { launchPhone, launchEmail } from '../../helpers/LaunchExternalApp';
-import { getSwedishMonthNameByTimeStamp } from '../../helpers/DateHelpers';
-import { Icon, Text } from '../../components/atoms';
-import { Card, HelpButton, ScreenWrapper, CaseCard } from '../../components/molecules';
-import { Modal, useModal } from '../../components/molecules/Modal';
-import BackNavigation from '../../components/molecules/BackNavigation';
-import Button from '../../components/atoms/Button';
-import { formatAmount, convertDataToArray, calculateSum } from '../../helpers/FormatVivaData';
-import AuthContext from '../../store/AuthContext';
-import { put } from '../../helpers/ApiRequest';
+import React, {
```

Comment:
This file is getting very big and not that easy to read. Can we break out some parts of it and make components out of it instead?

In my opinion, those components could exist in the caseScreens folder because they would in that case only be used in CaseSummary.js

Reply:
I believe we should break them out, but that it's outside the scope of this PR.

Reply:
If it is out of scope of this PR then I suggest that you create a task for it so we can track this improvement.

Reply:
I'll add a task for it.